_posts/workshops/python-open-science-tool-setup/2018-07-20-setup-04-setup-conda-earth-analytics-environment.md

```diff
@@ -169,21 +169,24 @@ For example, if a `Python` package such as `geopandas` is only installed in the
 
 ### Activate a Conda Environment
 
-**To active an environment**, use the Terminal to navigate to your earth-analytics directory (e.g. `cd` to the directory). Then, type the following command for your operating system to activate the environment (e.g. earth-analytics-python):
+**To activate an environment**, use the Terminal to navigate to your earth-analytics directory (e.g. `cd` to the directory). Then, type the following command to activate the environment (e.g. earth-analytics-python):
 
+```bash
+conda activate earth-analytics-python
+``` 
 
-#### Mac and Linux Instructions: 
+<a href="https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html" target="_blank">For older installations of Anaconda (versions prior to 4.6)</a>, type:
+
+#### Mac, Linux, and Git Bash (Windows) Instructions: 
 
 ```bash
 source activate earth-analytics-python
-
 ```
 
-#### Windows Instructions 
+#### Windows (Other Terminals) Instructions 
```

Comment:
@jlpalomino  is this for windows command prompt only? that is my understanding. it may be worth checkin in with Joe or someone on windows because if i remember correctly, it's most command prompt that is problematic on windows.

Reply:
Confirmed with Joe. Instructions have been updated to indicate these differences and the recommended terminal for Windows users.